crates/jit/src/unwind/systemv.rs

```diff
@@ -90,40 +90,40 @@ impl UnwindRegistry {
         let mut eh_frame = EhFrame(EndianVec::new(RunTimeEndian::default()));
         table.write_eh_frame(&mut eh_frame).unwrap();
 
-        // GCC expects a terminating "empty" length, so write a 0 length at the end of the table.
-        eh_frame.0.write_u32(0).unwrap();
+        if cfg!(all(target_os = "linux", any(target_env = "gnu", target_env = ""))) {
```

Comment:
AFAIK `target_env = ""` doesn't happen that often on Linux (I'm not sure it ever does), so that may be safe to drop here? Since this seems to be glibc-specific behavior for now it may be best to stick to just `target_env = "gnu"` anyway too

Reply:
Yeah, just going off the docs:

> Thus, for example, on many GNU platforms, this value will be empty

I'll drop it for now and merge.